### PR TITLE
New Hukamana API

### PR DESCRIPTION
**Pull request by bogas04**
Blockers;

* [x] API needs to update daily
* [ ] Handling multiple shabads sent by API.

### CODE REVIEW

**Comment by navdeepsinghkhalsa** [comment]
Unfortunately, I don't know TypeScript and the PR covers more than just Hukamnama, so I will defer back to you on it

File: src/js/pages/Hukamnama/index.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          gurbani: versesToGurbani(shabad.verses.map(v => ({ verse: v }))),
+          shabadinfo: shabadInfoMapper(shabad.shabadInfo),
+          navigation: {
+            previous: String(shabad.navigation.previous),
+            next: String(shabad.navigation.next),
+          },
+          source: shabad.shabadInfo.source.sourceId,
+        };
+
+        return <Layout data={layoutData} date={apiData.date} />;
```
Comment (bogas04) [inline src/js/pages/Hukamnama/index.tsx]: Can you check this part @navdeepsinghkhalsa, basically that I'm getting the data correctly. Right now I only pick shabads[0]

Comment (tarunsingh5) [reply src/js/pages/Hukamnama/index.tsx]: @navdeepsinghkhalsa can you ptal

**Comment by tarunsingh5** [comment]
bump

**Comment by bogas04** [comment]
@Gauravjeetsingh do you think we can use this code or your new PR makes this irrelevant?

Thanks!

**Comment by Gauravjeetsingh** [comment]
@bogas04 Yeah, initially I tried to use it, but these changes break the things we fixed recently.
So, I am already cherry picking things from here, for the new PR. I will look into typescript thing as well. Thanks!

**Comment by bogas04** [comment]
Is this obsolete now @Gauravjeetsingh ?

**Comment by Gauravjeetsingh** [comment]
@bogas04 ji yes, this is obsolete now. We already merged the newer hukamnama API in production.

**Pull request closed without merging**